src/test/java/com/dataStructures/HashTableTest.java

```diff
@@ -0,0 +1,75 @@
+package src.test.java.com.dataStructures;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ *
+ * @author Rick
+ */
+public class HashTableTest {
+    
+    /**
+     * Test of add method, of class HashTable.
+     */
+    @org.junit.Test
+    public void testAdd() {
+        System.out.println("add");
```

Comment:
`println` statements are unnecessary in unit tests.

Reply:
solved